projects/payment-lib/src/lib/services/xl-file/xl-file.service.ts

```diff
@@ -169,6 +174,17 @@ export class XlFileService {
     return worksheet;
   }
 
+    private setTelphonyPaymentsReportHeaders (worksheet: ExcelJS.Worksheet): ExcelJS.Worksheet {
+      worksheet.getCell('A1').value = "Resp_Service Name";
```

Comment:
I asked Ali to clean up these headings. Is there any reason to do this, just looks unproffessional. What about
      worksheet.getCell('A1').value = "Service Name";
      worksheet.getCell('B1').value = "CCD Reference";

You'll have to change line 38 and all other references to these heading as well if you do change.

Reply:
As specified by Ali yesterday these are what he said

Reply:
Ok, we'll go with these headings. Thanks.